include/mesh.cc

```diff
@@ -0,0 +1,88 @@
+//! Create a nodal pointer based on input id and coordinates and check whether
+//! duplicate nodes exist
+void Mesh::create_node(std::vector<unsigned> id,
```

Comment:
Give a starting unique id, and increment automatically, no need for a vector of ints